v2/yarpcthrift/thriftrw-plugin-yarpcv2/roundtrip_test.go

```diff
@@ -243,50 +233,35 @@ func testRoundTrip(t *testing.T, enveloped, multiplexed bool) {
 			methodArgs:    []interface{}{ptr.String("foo")},
 			wantError:     &atomic.KeyDoesNotExist{Key: ptr.String("foo")},
 		},
-		{
-			desc:          "store: forget",
-			procedures:    storeserver.New(&storeHandler{}, serverOpts...),
-			newClientFunc: storeclient.New,
-			method:        "Forget",
-			methodArgs:    []interface{}{ptr.String("foo")},
-			wantAck:       true,
-		},
-		{
-			desc:          "store: forget error",
-			procedures:    storeserver.New(&storeHandler{failWith: errors.New("great sadness")}, serverOpts...),
-			newClientFunc: storeclient.New,
-			method:        "Forget",
-			methodArgs:    []interface{}{ptr.String("foo")},
-			wantAck:       true,
-		},
 	}
 
 	ctx := context.Background()
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
-			httpInbound := http.NewTransport().NewInbound(":0")
-
-			server := yarpc.NewDispatcher(yarpc.Config{
-				Name:     "roundtrip-server",
-				Inbounds: yarpc.Inbounds{httpInbound},
-			})
-			server.Register(tt.procedures)
-			require.NoError(t, server.Start())
-			defer server.Stop()
-
-			outbound := http.NewTransport().NewSingleOutbound(
-				fmt.Sprintf("http://%v", yarpctest.ZeroAddrToHostPort(httpInbound.Addr())))
-
-			dispatcher := yarpc.NewDispatcher(yarpc.Config{
-				Name: "roundtrip-client",
-				Outbounds: yarpc.Outbounds{
-					"roundtrip-server": {
-						Unary: outbound,
-					},
-				},
-			})
-			require.NoError(t, dispatcher.Start())
-			defer dispatcher.Stop()
+			router := yarpcrouter.NewMapRouter("roundtrip-server")
+			router.Register(tt.procedures)
+			listener, err := net.Listen("tcp", ":0")
+			require.NoError(t, err)
+			httpInbound := &yarpchttp.Inbound{
```

Comment:
let's just call this `inbound` so we have symmetry with the `outbound` variable below